electrochemistry.ttl

```diff
@@ -1297,13 +1402,16 @@ Aluminum foil"""@en ;
  ] .
 
 
+###  http://emmo.info/electrochemistry#electrochemistry_2dd44ff6_425a_4377_b86e_fa2bd567819f
+:electrochemistry_2dd44ff6_425a_4377_b86e_fa2bd567819f rdf:type owl:Class ;
+                                                       rdfs:subClassOf :electrochemistry_0a03ce7e_d79f_412e_a103_b5d74de9f4d7 ;
+                                                       emmo:EMMO_967080e5_2f42_4eb2_a3a9_c58143e835f9 "Rapid, transient change in the amplitude of an electric potential, from a baseline value to a higer or lower value, followed by a rapid return to the baseline value."@en ;
+                                                       skos:prefLabel "PotentialPulse"@en .
+
+
 ###  http://emmo.info/electrochemistry#electrochemistry_2dea31c2_5061_4464_ab76_5336bef23629
 :electrochemistry_2dea31c2_5061_4464_ab76_5336bef23629 rdf:type owl:Class ;
-                                                       rdfs:subClassOf emmo:EMMO_0f6f0120_c079_4d95_bb11_4ddee05e530e ,
-                                                                       [ rdf:type owl:Restriction ;
-                                                                         owl:onProperty emmo:EMMO_5d73661e_e710_4844_ab9b_a85b7e68576a ;
-                                                                         owl:someValuesFrom emmo:EMMO_66bc9029_f473_45ff_bab9_c3509ff37a22
-                                                                       ] ;
+                                                       rdfs:subClassOf emmo:EMMO_0f6f0120_c079_4d95_bb11_4ddee05e530e ;
                                                        skos:prefLabel "CelsiusTemperatureMeasurementResult"@en .
```

Comment:
This terms seems very specific. I would argue that MeasurementResult is good enough, no need to specify Temperature nor Celsius. That distinction can be done in JSON-LD: 
```
"@type": "MeasurementResult",
"hasMeasuredProperty": {
   "@type":"AmbientThermodynamicTemerature",
   "hasReferenceUnit":"Celsius"
   }
```